### PR TITLE
restructure docs

### PR DESCRIPTION
**Pull request by m-kharbat**
screen shot 2017-03-01 at 16 48 16
screen shot 2017-03-01 at 16 48 23

### CODE REVIEW

File: blackbox/docs/sql/ddl/system_columns.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.. _sql_ddl_system_columns:
+
+System Columns
+==============
```
Comment (dobe) [inline blackbox/docs/sql/ddl/system_columns.txt]: this is not DDL

File: blackbox/docs/sql/ddl/sharding.txt [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.. _sql_ddl_sharding:
+
+Sharding
+========
```
Comment (dobe) [inline blackbox/docs/sql/ddl/sharding.txt]: pls add introduction here

File: blackbox/docs/sql/administration/set_reset.txt [hunk 3]
```diff
@@ -0,0 +1,96 @@
+.. _ddl-set-reset:
```
Comment (dobe) [inline blackbox/docs/sql/administration/set_reset.txt]: no ddl

File: blackbox/docs/sql/ddl/sharding.txt (hunk 2 above)
Comment (nslater) [inline blackbox/docs/sql/ddl/sharding.txt]: added. pls review

**Pull request merged**